easybuild/easyconfigs/a/Automake/Automake-1.15-GCCcore-5.4.0.eb

```diff
@@ -22,11 +22,15 @@ toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7946e945a96e28152ba5a6beb0625ca715c6e32ac55f2e353ef54def0c8ed924']
 
 dependencies = [('Autoconf', '2.69')]
 
 # use same binutils version that was used when building GCCcore toolchain
-builddependencies = [('binutils', '2.26', '', True)]
+builddependencies = [
+    ('binutils', '2.26', '', True),
```

Comment:
Hmm, I see no reason to hardcode to use a `dummy` install of `binutils` here, maybe we should fix that too while we're at it?

Reply:
Yeah, that is another thing we should be more consistent about

Reply:
We usually don't hardcode to `dummy` toolchain, but there are cases we need to, to avoid cyclic dependencies.